autoload/go/import.vim

```diff
@@ -161,8 +164,15 @@ function! go#import#SwitchImport(enabled, localname, path, bang) abort
         let linesdelta += 3
         let appendstr = qlocalpath
         let indentstr = 1
+      elseif getline(appendline) =~# '^import\s\+(\+)'
+        call setline(appendline, 'import (')
+        call append(appendline + 0, appendstr)
+        call append(appendline + 1, ')')
+        let linesdelta -= 1
+        let indentstr = 1
+      else
+        call append(appendline, appendstr)
```

Comment:
The `appendline == packageline` needs to call `append(appendline, appendstr)` now, too.

Reply:
Thanks for the catch, fixed it.